mysql-test/suite/rpl/t/rpl_bug77684.test

```diff
@@ -0,0 +1,33 @@
+# Test case for bug #77684 (DROP TABLE IF EXISTS may brake replication if slave has replication filters)
+--source include/master-slave.inc
+
+# Test A (using fully-qualified table names) 
+--connection master
+CREATE DATABASE db1;
+
+CREATE TABLE db1.t1a (ID bigint(20) primary key) ENGINE=InnoDB;
```

Comment:
The testcase needs "--source include/have_innodb.inc" at the top too (since it's dependent on InnoDB, being the only FK-supporting engine in MySQL)